account_move_tier_validation/tests/test_tier_validation.py

```diff
@@ -66,25 +69,25 @@ def test_03_move_post(self):
                 "reviewer_id": self.test_user_1.id,
             }
         )
+
         partner = self.env["res.partner"].create({"name": "Test Partner"})
         product = self.env["product.product"].create({"name": "Test product"})
+
         invoice = self.env["account.move"].create(
             {
                 "move_type": "out_invoice",
                 "partner_id": partner.id,
-                "invoice_date_due": fields.Date.from_string("2024-01-01"),
+                "invoice_date_due": fields.Date.to_date("2024-01-01"),
                 "invoice_line_ids": [
                     (0, 0, {"product_id": product.id, "quantity": 1, "price_unit": 30})
                 ],
             }
         )
-        invoice.with_user(self.test_user_2.id).request_validation()
+        reviews = invoice.with_user(self.test_user_2.id).request_validation()
+        self.assertTrue(reviews)
+        with self.assertRaises(ValidationError):
+            invoice.action_post()
         invoice = invoice.with_user(self.test_user_1.id)
-        invoice.invalidate_model()
         invoice.validate_tier()
-        with self.assertRaisesRegex(
```

Comment:
You are removing a meaningful test here. Is it no longer necessary to pass skip_validation_check in `action_post` (in which case that override can be removed), or have the internals of `action_post` changed so that the test needs to be updated?